idris-tests.el

```diff
@@ -157,6 +157,17 @@ remain."
       (kill-buffer))
     (idris-quit)))
 
+(ert-deftest idris-test-idris-type-search ()
+  "Test that `idris-type-search' produces output in Idris info buffer."
```

Comment:
unfortunately this test does not test the regression as I wasn't able  replicate the error in `elisp` 
using `funcall-interactively` or `call-interactively`

Reply:
Okay, you can submit a separate PR with the regression test. I wish to add the revert ASAP.